src/controllers/countries.js

```
@@ -0,0 +1,32 @@
+    res.json(countries);
+  } catch (error) {
+    console.error(error);
+    res.status(500).json({ message: 'Error with getting countries' });
+  }
+};
+
+const fetchCitiesByCountry = async (req, res) => {
+  const { countryCode } = req.params;
+
+  if (!countryCode) {
+    return res.status(400).json({ message: 'Country code is required' });
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)